app/src/renderer/vuex/modules/user.js

```diff
@@ -38,9 +38,18 @@ export default ({}) => {
   }
 
   const mutations = {
+    setSignIn(state, hasSignedIn) {
+      state.signedIn = hasSignedIn
+    },
     setAccounts(state, accounts) {
       state.accounts = accounts
     },
+    setUserAddress(state, address) {
+      state.address = address
+    },
+    setAtoms(state, atoms) {
+      state.atoms = atoms
```

Comment:
is there a reason we're not using `bondingDenom` here?

Reply:
`setAtoms` should be removed as we only use `liquidAtoms` now 🤷‍♂️